app/helpers/account/profiles_helper.rb

```diff
@@ -0,0 +1,5 @@
+module Account::ProfilesHelper
+  def profile_avatar(user)
+    image_tag((user.with_avatar? ? (url_for user.avatar) : ("128x128.png")),  class: 'is-rounded')
```

Comment:
why do you need so more brackets?

Reply:
it's coz code style is inconsistent.. he's using methods w/ arg w/o brackets and strings in brackets..
like: `url_for user.avatar : "128x128.png"` would cause a syntax error..

Reply:
instead of (url_for user.avatar) - wrie url_for(user.avatar)